src/new_versions.jl

```diff
@@ -79,6 +82,18 @@ function make_pr_for_new_version(api::GitHub.GitHubAPI,
               latest_version,
               name,
               dep)
+    elseif !bump_compat_containing_equality_specifier &&
+            !isnothing(current_compat_entry_verbatim) &&
+            contains(current_compat_entry_verbatim, '=') &&
+            !contains(current_compat_entry_verbatim, '>') &&
+            !contains(current_compat_entry_verbatim, '<')
```

Comment:
@DilumAluthge I'm not sure if `current_compat_entry_verbatim` will contain the entire compat entry line (e.g., `FFTW = "=1.2.2"`). If that is the case, this needs to be changed.

Reply:
`current_compat_entry_verbatim` would only be e.g. `"=1.2.2"`. So this should be fine.